x-pack/plugins/apm/public/components/shared/service_link.tsx

```diff
@@ -35,22 +52,61 @@ export function ServiceLink({
     ? '/mobile-services/{serviceName}/overview'
     : '/services/{serviceName}/overview';
 
-  return (
-    <StyledLink
-      data-test-subj={`serviceLink_${agentName}`}
-      href={link(serviceLink, {
-        path: { serviceName },
-        query,
-      })}
-    >
-      <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+  if (serviceName === OTHER_SERVICE_NAME) {
+    return (
+      <EuiFlexGroup alignItems="center" gutterSize="xs">
         <EuiFlexItem grow={false}>
-          <AgentIcon agentName={agentName} />
+          <EuiText
+            grow={false}
+            style={{ fontStyle: 'italic', fontSize: '1rem' }}
+          >
+            _other
+          </EuiText>
         </EuiFlexItem>
-        <EuiFlexItem className="eui-textTruncate">
-          <span className="eui-textTruncate">{serviceName}</span>
+        <EuiFlexItem>
+          <PopoverTooltip
+            ariaLabel={i18n.translate('xpack.apm.serviceLink.tooltip', {
+              defaultMessage: 'Max service groups reached tooltip',
+            })}
+            iconType="alert"
+          >
+            <EuiText style={{ width: `${unit * 28}px` }} size="s">
+              <FormattedMessage
+                defaultMessage="The maximum number of unique services has been reached. Please increase {codeBlock} in APM Server."
+                id="xpack.apm.serviceLink.tooltip.message"
+                values={{
+                  codeBlock: <EuiCode>aggregation.service.max_groups</EuiCode>,
```

Comment:
What about users that are using Elastic Agent? Is this the name of the setting in the APM integration?

Reply:
I am still new and learning things everyday, may be my reply [above](https://github.com/elastic/kibana/pull/148641#discussion_r1073269845) helps answer this question.

Reply:
@achyutjhunjhunwala Can you forward this question to APM Server team? They'd know if the setting is configurable through fleet and what it's called.

Reply:
Yeah, let's ask the APM Server team. I'm also curious to know if this is supported in ESS. I don't see the setting listed in https://www.elastic.co/guide/en/cloud/current/ec-manage-apm-settings.html.

Reply:
To provide more insight into my thought process, @achyutjhunjhunwala: I'm hesitant to list `aggregation.service.max_groups` in the UI because I think that setting only applies to users running APM Server standalone. Users on ESS will have a different setting (if it exists) that is usually `apm-server.*`, and users with the APM integration will have a different setting (if it exists) that follows a different naming convention. 

Instead of calling out one of the three settings name in the UI, I think it makes more sense to link to a section of the Kibana docs where we can explain all three potential fixes for all three APM Server use cases.

Reply:
@bmorelli25 @sqren  Thank you for clarifying this. I will check with the APM Server team on this and get back

Reply:
I think we decided not to refer to that setting because users cannot change it. All they can do is increase memory afaiu.